lanes/free/README.md

```diff
@@ -0,0 +1,129 @@
+# 🏗️ Free Lane Setup
+
+## 📦 Dependencies
+
+The Block SDK is built on top of the Cosmos SDK. The Block SDK is currently
+compatible with Cosmos SDK versions greater than or equal to `v0.47.0`.
+
+## 📥 Installation
+
+To install the Block SDK, run the following command:
+
+```bash
+$ go install github.com/skip-mev/block-sdk
```

Comment:
I see packages in the repo defined as `skip-mev/pob` -> `skip-mev/block-sdk`

Reply:
going to rename the repo soon,